src/Equinor.ProCoSys.IPO.Command/InvitationCommands/EditInvitation/EditInvitationCommandHandler.cs

```diff
@@ -139,6 +139,11 @@ private async Task<List<McPkg>> GetMcPkgScopeAsync(IList<string> mcPkgNos, strin
                     throw new IpoValidationException("Could not find all mc pkgs in scope.");
                 }
 
+                if (mcPkgsFromMain.Any(mc => mc.OperationHandoverStatus == "Accepted"))
```

Comment:
same comment here as in CreateInvitationCommandHandler.cs regarding throwing exception